client-side/src/components/signUp/PasswordStrength.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, LinearProgress } from '@mui/material';
+import './PasswordStrength.scss';
```

Comment:
reorder the imports, style files should be the last ones

Reply:
קשור לאפרת הרטמן